test/image-blueprints/layer5-bootc/group1/cos9-bootc-source.containerfile

```diff
@@ -0,0 +1,59 @@
+FROM quay.io/centos-bootc/centos-bootc:stream9
+
+# Build arguments
+ARG PREVIOUS_MINOR_VERSION=15
+ARG MICROSHIFT_REPO_NAME=microshift-local
```

Comment:
May I suggest including `RPM` in the name? I was really wondering why we want to copy `openshift/microshift` to the container :)